src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -289,6 +290,7 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
                 case MessageID.IDS_DefaultInterfaceImplementation: // semantic check
                 case MessageID.IDS_BaseTypeInBaseExpression:
                 case MessageID.IDS_OverrideWithConstraints: // semantic check
+                case MessageID.IDS_FeatureAllowGenericAttributes:
```

Comment:
>IDS_FeatureAllowGenericAttributes [](start = 31, length = 33)

Add `// semantic check, PROTOTYPE(generic-attributes) need to attach feature to the proper language version once we have one`

Reply:
Done